gaphor/diagram/drop.py

```diff
@@ -119,6 +120,27 @@ def drop_relationship_on_diagram(element: Relationship, diagram: Diagram, x, y):
     )
 
 
+@drop.register(ActivityEdge, Diagram)
```

Comment:
Hi @pbrown12303, thanks so much for the contribution and I really like that you included tests to go along with it!

Since this new function almost exactly repeats the one above it, did you consider stacking the `@drop.register` decorators? You should be able to put one on top of the other. If you then update the type hint for the `element` function argument to be `Relationship | ActivityEdge` it should be able to take either type of element as an input to the function and simplify the code to use just a single function.

Reply:
Dan,

I thought of that, but when the Dependency is selected in the tree (as
opposed to the diagram) there is no DependencyItem, which is diagram
specific. There are DiagramItem properties (i.e. the Automatic setting)
displayed in the DiagramItem properties ui. For this reason I had to
separate them.